tensorflow/python/tools/import_pb_to_tensorboard.py

```diff
@@ -29,6 +29,13 @@
 from tensorflow.python.platform import gfile
 from tensorflow.python.summary import summary
 
+# Try importing TensorRT ops if available
+# pylint: disable=unused-import,trailing-whitespace,g-import-not-at-top,wildcard-import
+try:
+  from tensorflow.contrib.tensorrt.ops.gen_trt_engine_op import *
```

Comment:
I may miss something, as I tried to run tensorboard to visualize a graph with trt ops and it works, so what's the purpose of importing this?

Reply:
@aaroey, It fails with unknown op type if I try to run it on a frozen graph with TRTEngineOps.

Reply:
I think a better solution will be something like:

1. add `from tensorflow.contrib import tensorrt` to `tensorflow/contrib/__init__.py`
2. add dependency `"//tensorflow/contrib:contrib_py"` to the build rule of this file in `tensorflow/python/tools/BUILD`
3. add `from tensorflow.contrib import *` to this file

But there is a problem with the `import tensorrt` in `contrib/__init__.py`: 
```
  File "...org_tensorflow/tensorflow/contrib/tensorrt/wrap_conversion.py", line 24, in swig_import_helper
    _mod = imp.load_module('_wrap_conversion', fp, pathname, description)
ImportError: dynamic module does not define init function (init_wrap_conversion)
```

So this could be a TODO. Would you help to add the TODO?
Also could you also add the build dependency to the BUILD rule of this file (step2)?